src/App.jsx

```diff
@@ -1,60 +1,178 @@
-import "./App.css"
+import { useState } from "react";
+import "./App.css";
 
 function App() {
+  const [firstNumber, updateFirstNumber] = useState(0);
+  const [operation, updateOperation] = useState("+");
+  const [secondNumber, updateSecondNumber] = useState(0);
+  const [result, setResult] = useState(0);
+
+  const ClearFirst = () => {
+    // Description: Sets number to 0 on click "clear"
+    console.log(`Cleared first`);
+    // updateNumber to 0
+    updateFirstNumber(firstNumber * 0);
+  };
+  const ClearSecond = () => {
+    // Description: Sets number to 0 on click "clear"
+    console.log(`Cleared second`);
+    // updateNumber to 0
+    updateSecondNumber(secondNumber * 0);
+  };
+
+  const SetfirstNumber = (event) => {
+    // Description: Sets number to provided number on click of number button
+    // Input: event
+    console.log(`New firstNumber: ${event.currentTarget.value}`);
+    // Check if firstNumber is 0, if 0, set to button value, otherwise add onto
+    // Turn firstNumber into string for easier comparison
+    const nr = firstNumber.toString();
+    if (nr === "0") {
+      updateFirstNumber(event.currentTarget.value);
+    } else {
+      // updateNumber to value on button (add onto)
+      updateFirstNumber(firstNumber + event.currentTarget.value);
+    }
+  };
+  const SetOperation = (event) => {
+    // Description: Sets number to provided number on click of number button
+    // Input: event
+    console.log(`New operation: ${event.currentTarget.value}`);
+    // updateOperaton to value on button
+    updateOperation(event.currentTarget.value);
+  };
+  const SetSecondNumber = (event) => {
+    // Description: Sets number to provided number on click of number button
+    // Input: event
+    console.log(`New secondNumber: ${event.currentTarget.value}`);
+    // Check if secondNumber is 0, if 0, set to button value, otherwise add onto
+    // Turn secondNumber into string for easier comparison
+    const nr = secondNumber.toString();
+    if (nr === "0") {
+      updateSecondNumber(event.currentTarget.value);
+    } else {
+      // updateNumber to value on button (add onto)
+      updateSecondNumber(secondNumber + event.currentTarget.value);
+    }
+  };
+
+  const Answer = () => {
```

Comment:
better to name this `computeAnswer` since the name includes the action of what the function does